tiatoolbox/data/pretrained_model.yaml

```
@@ -611,6 +611,31 @@ fcn_resnet50_unet-bcss:
             stride_shape: [256, 256]
             save_resolution: {'units': 'mpp', 'resolution': 0.25}
 
+unet_tissue_mask_tsef:
+    url: https://tiatoolbox.dcs.warwick.ac.uk/models/registration/UNET-v0.0-TSEF.pth
+    description:
+      These weights can be used to segment a grayscale image to
+      create a tissue mask excluding the fatty region. TSEF stands for tissue
+      segmentation excluding fat.
```

```suggestion
    description: >
      These weights can be used to segment a grayscale image to
      create a tissue mask excluding the fatty region. TSEF stands for tissue
      segmentation excluding fat.
```

Add > to mark that new lines should be replaced with spaces when parsed and a new line added at the end.